reachy_sdk/head.py

```diff
@@ -30,13 +30,17 @@ class Head:
         'neck_disk_bottom', 'neck_disk_middle', 'neck_disk_top',
     }
 
+    _antennas = {'l_antenna', 'r_antenna'}
```

Comment:
Why not directly adding them to required_joints?

Reply:
Because required_joints is used for the disk property and to compute the look_at.

Reply:
Only disk_joints is used no, not required_joints?